### DISCUSSION TITLE
How can I transfer some other information with the flow from ue to gnb to upf

### DISCUSSION THREAD

**Discussion by MrPluto0** [srsran/srsRAN_Project#667] [General Help]
I am doing an experiment. I want to transfer some other information with the flow from ue to gnb to upf. Such as, i want a dataset transfered to gnb's Server by the network flow. And the gnb server can train or do other things with the dateset.
Is there some ways to do so? Thanks!

**Comment by pgawlowicz** [comment]
@MrPluto0 any update on this idea?
